packages/p2p/src/components/order-details/__test__/order-details.spec.js

```diff
@@ -80,11 +111,26 @@ describe('<OrderDetails/>', () => {
         useStores.mockReturnValue({
             order_store: {
                 ...mock_order_store,
-                order_information: { ...mock_order_info, should_show_lost_funds_banner: true, chat_channel_url: null },
+                order_information: {
+                    ...mock_order_info,
+                    should_show_lost_funds_banner: true,
+                    chat_channel_url: null,
+                },
             },
             sendbird_store: { ...mock_sendbird_store },
+            my_profile_store: { ...mock_my_profile_store },
+            buy_sell_store: { ...mock_buy_sell_store },
+            general_store: { ...mock_general_store },
+        });
+        useModalManagerContext.mockImplementation(() => ({
+            isCurrentModal: false,
+            showModal: () => {},
+            hideModal: () => {},
+            useRegisterModalProps: () => {},
+        }));
```

Comment:
This is repetitive. Can we return the value on line 93 instead of mocking it in every test?
```
jest.mock('Components/modal-manager/modal-manager-context', () => ({
    ...jest.requireActual('Components/modal-manager/modal-manager-context'),
    useModalManagerContext: jest.fn(() => ({
            isCurrentModal: false,
            showModal: () => {},
            hideModal: () => {},
            useRegisterModalProps: () => {},
     })),
}));
```